web-server/src/components/Teams/useTeamsConfig.tsx

```diff
@@ -263,7 +263,18 @@ export const TeamsCRUDProvider: React.FC<{
             variant: 'success',
             autoHideDuration: 2000
           });
-          fetchTeamsAndRepos();
+          const createdTeam = res.payload.team;
+
+          fetchTeamsAndRepos().then((res: any) => {
+            if (res.meta.requestStatus === 'fulfilled') {
+              const { teams } = res.payload;
+              const singleTeam = teams.find(
+                (team: Team) => team.id === createdTeam.id
+              );
+              dispatch(appSlice.actions.setSingleTeam([singleTeam]));
```

Comment:
this check will never fail, but still,
`if(singleTeam)`

Reply:
Got it